extensions/sql-migration/src/wizard/skuRecommendation/assessmentSummaryCard.ts

```diff
@@ -393,20 +407,47 @@ export class AssessmentSummaryCard implements vscode.Disposable {
 			}
 		});
 
+		this._saveTemplateLink = view.modelBuilder.hyperlink().withProps({
+			label: constants.TARGET_PROVISIONING_TITLE,
+			url: '',
+			height: 18,
+			CSSStyles: {
+				'font-size': '13px',
+				'font-weight': '400',
+				'line-height': '18px',
+				'text-decoration': 'none',
+				'margin-right': '15px',
+				'display': 'none',
+			}
+		}).component();
+
+		this._saveTemplateLink.onDidClick(async () => {
+			if (hasRecommendations(this.migrationStateModel)) {
+				const generateProvisioningScriptDialog = new GenerateProvisioningScriptDialog(this.migrationStateModel, this.migrationTargetType);
```

Comment:
Use Try catch here to catch any error, and log it. It will be helpful while debugging.